cfme/tests/configure/test_analysis_profiles.py

```diff
@@ -333,3 +336,77 @@ def test_analysis_profile_description_validation(analysis_profile_collection):
     )
     view.flash.assert_message("Description can't be blank")
     view.cancel.click()
+
+
+@test_requirements.rbac
+@pytest.mark.tier(1)
+@pytest.mark.ignore_stream('5.9')
+@pytest.mark.parametrize(
+    'product_features',
+    [  # Navigation for product features trees
```

Comment:
Optional:
I think we should put this as global veritable not like this.... and then parametrize. It looks like mess in test pedestrianization. its my personal view.

Reply:
Totally agree here Nikhil, was going to make the same comment.

Reply:
I want to say make this **Required** since we both feel the same here

Reply:
@jawatts @digitronik 
I have created global list and then parametrised it.
Thank for suggestion.